tests/conftest.py

```diff
@@ -116,3 +119,33 @@ def virtual_drive():
     virtual_servo = net.connect_to_slave(server.ip, server.dictionary_path, server.port)
     yield server, virtual_servo
     server.stop()
+
+
+@pytest.fixture(scope="session")
+def connect_to_rack_service():
+    rack_service_port = 33810
+    client = rpyc.connect("localhost", rack_service_port, config={"sync_request_timeout": None})
```

Comment:
I've put some documentation relating to this: https://novantamotion.atlassian.net/wiki/spaces/TEST/pages/791445537/How+to+use+Rack+Service